packages/format-library/src/index.js

```diff
@@ -8,18 +15,21 @@ import { italic } from './italic';
 import { link } from './link';
 import { strikethrough } from './strikethrough';
 
-/**
- * WordPress dependencies
- */
-import {
-	registerFormatType,
-} from '@wordpress/rich-text';
+export function getCoreFormatTypes() {
+	return [
+		bold,
+		code,
+		image,
+		italic,
+		link,
+		strikethrough,
+	].map(
+		( { name, ...settings } ) => [ name, settings ]
```

Comment:
Why did we land on an array return value here.

Reply:
> The shape it returns would have to be defined, I opted for the simplicity of refactoring :)

We can make it `{ name, settings }` or reduce everything to `{ [ name ]: settings, ... }`. I'm fine with all of them.